resource/resources.go

```diff
@@ -11,4 +11,9 @@ const (
 	ResourceNameDisk          = "ephemeral-storage"
 	ResourceNameDiskLegacy    = "storage"
 	ResourceNamePods          = "pods"
+	ResourceNameMockNodesPool = "mock-nodes"
```

Comment:
This shouldn't go in resources.go, wrong kind of `resource`.

make `resourcepools/resourcepools.go`

Reply:
making resourcepool (keeping singular to match pod and node)